app/uk/gov/hmrc/fileupload/controllers/FileController.scala

```diff
@@ -26,7 +26,7 @@ import uk.gov.hmrc.fileupload._
 import uk.gov.hmrc.fileupload.infrastructure.BasicAuth
 import uk.gov.hmrc.fileupload.read.envelope.{Envelope, WithValidEnvelope}
 import uk.gov.hmrc.fileupload.read.file.Service._
-import uk.gov.hmrc.fileupload.write.envelope.{EnvelopeCommand, EnvelopeNotFoundError, FileNotFoundError, StoreFile}
```

Comment:
tbh, I still prefer explicit imports, but maybe this is a stylistic thing. Not a showstopper.